root/etc/e-smith/templates/etc/sudoers.d/50_servermanager_perms/20GroupToCmndAlias

```diff
@@ -0,0 +1,81 @@
+{
+    use strict;
+    use warnings;
+    use JSON;
+
+    require "/usr/libexec/nethserver/api/lib/auth_functions.pl";
+
+    # Following the user's permissions 
+    # we could not be granted to the esmith API
+    my $groupAdmins = `sudo /sbin/e-smith/db configuration getprop admins group`;
```

Comment:
Here the "internal" sudo call must be removed. The template expansion already occurs with root privileges.

Reply:
IIRC when the user is not root, I could not use the esmith API. We need more test here, it works already like this for the actual sudo delegation

Reply:
We have to change the way `system-authorization/read` is invoked in many places:

https://github.com/NethServer/nethserver-cockpit/search?l=Vue&q=system-authorization%2Fread

There must be set `superuser = true` in [exec() calls](https://github.com/NethServer/nethserver-cockpit/blob/5d1444c722a08a8554100ab9db9bbb7e16bdcaf5/root/usr/share/cockpit/nethserver/libs/nethserver.js#L3)